lib/web3.js

```diff
@@ -78,56 +79,60 @@ var setupProperties = function (obj, properties) {
 };
 
 /// setups web3 object, and it's in-browser executed methods
-var web3 = {};
-web3.providers = {};
-web3.currentProvider = null;
-web3.version = {};
-web3.version.api = version.version;
-web3.eth = {};
+function Web3(provider) {
```

Comment:
Yeah, our ide works with 2 and more JSON RPC in parallel.

Okay, I see the problem. Yeah, that could be unnecessary complication. But if you think it's useful to work with multiple providers, you could get rid of `RequestManager` singleton and current global `web3` and define global object like `window.web3 = new Web3()`.